demo/start.sh

```diff
@@ -81,10 +81,13 @@ for port in 8080 8081 8082; do
             echo "tls_private_key_path: \"$DIR/$port/localhost:$port.tls.key\""
 
             # Ignore keys from the trusted keys server
-            echo '# Ignore keys from the trusted keys server'
+            echo '# Ignore keys from the trusted keys server.'
+            echo '# Specifically, we do this by omitting the "verify_keys" option'
+            echo '# and enabling "accept_keys_insecurely"'
             echo 'trusted_key_servers:'
             echo '  - server_name: "matrix.org"'
             echo '    accept_keys_insecurely: true'
+            echo 'suppress_key_server_warning: true'
```

Comment:
Would it not make more sense to not use a trusted key server at all in the demo script?

Reply:
I was tempted to do so, but figured not to rock the boat.

However - upon testing `trusted_key_servers: []` (not setting the option [defaults to matrix.org](https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html#trusted_key_servers)), it works as expected :tada: 

I've updated the config to simple request signing keys directly from the server in c8d0f97e10184a3f94eb0fff3f1da6b65e2efb52. Thanks for calling it out!